libs/shared/ui/src/lib/components/slider/slider.tsx

```diff
@@ -0,0 +1,57 @@
+import { useRef, useState } from 'react'
+
+export interface SliderProps {
+  min: number
+  max: number
+  step: number
+  defaultValue?: number
+  label?: string
+  valueLabel?: string
+  className?: string
+  getValue?: (value: number) => void
+}
+
+export function Slider(props: SliderProps) {
+  const { min, max, step, defaultValue, label, valueLabel, className = '', getValue } = props
+  const [value, setValue] = useState(defaultValue || min)
+
+  const inputEl = useRef(null)
+
+  const handleChange = (value: string) => {
+    const valueInt = parseInt(value, 10)
+    setValue(valueInt)
+    if (getValue) getValue(valueInt)
+  }
+
+  return (
+    <div className={`slider w-full ${className}`}>
+      {label && (
+        <div className="flex justify-between items-center mb-2">
+          {label && <p className="text-text-500 text-sm">{label}</p>}
+          {valueLabel && (
+            <p className="text-brand-500 font-medium">
+              {value}
+              {valueLabel}
+            </p>
+          )}
+        </div>
+      )}
+      <input
+        style={{
+          background: `linear-gradient(to right, #5b50d6 0%, #5b50d6 ${((value - min) / (max - min)) * 100}%, #C6D3E7 ${
+            ((value - min) / (max - min)) * 100
+          }%, #C6D3E7 100%)`,
```

Comment:
There is color `#C6D3E7` that is written in static, maybe we should have a place where all colors are agregated to avoir changes in the future ?

Reply:
Yes I agree, having a constant with all colors variables could be nice
I'll look to work on that, I noted that on my todo